### PR TITLE
Update CAPP Agile Learning product page to 2019

### LINKED ISSUE #1939: Update Agile Learning product page

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This introduces some cool new features too: 

- On page navigation partial with local variables
- Standardised product page layout which partly targets: #1762

## Todos:

- [x] Introduce new logo in hero
- [x] Update hero (image)
- [x] Update navigation item
- [x] Update product overview item
- [x] Update homepage mention of CAPP Agile Learning solution
- [x] Update H1 ~and H2 texts~
- [x] Remove demo button
- [x] Update main content
- [x] Add new productsheet for downloading
- [x] Introduce Product Layout

Close #1939

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Deploy preview: 

https://deploy-preview-1940--defacto-nl.netlify.com/capp-agile-learning/

Check homepage and main navigation too!

File: source/localizable/capp-agile-learning/new/_product-overview.nl.erb [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        Maak & deel leercontent
+      </h2>
+      <p>
+        Maken en delen van <strong>zelfgemaakte en bestaande</strong> (leer)content.
+      </p>
+      <p>
+        Zorg voor ordening met <strong>hoofdstukken</strong> en bundel gelijksoortige onderwerpen in <strong>leerpaden</strong>.
+      </p>
+      <p>
+        <strong>Slimme editor</strong> voor opmaak en insluiten video's van YouTube en online afbeeldingen.
```
Comment (sn3p) [inline source/localizable/capp-agile-learning/new/_product-overview.nl.erb]: "online afbeeldingen" ? 🤔

**Comment by sn3p** [comment]
Looks good ❤️  Just a few suggestions:

I'm missing some hierarchy and coherency in the content, it's a lot of summing up USP's at the moment.

The introduction text could be a bit larger in font-size maybe, making it more important.
I'm not sure about the 1.2.3. summing up (presentation wise). Maybe integrate this in the text? Or merge this with the sections itself?

Just thinking out loud, feel free to ignore :)

---

Some inline hacking in the inspector:

deploy-preview-1940--defacto-nl netlify com_capp-agile-learning copy

**Comment by sn3p** [comment]
Would be cool to make the subnav sticky 😎 

Screenshot 2019-07-11 at 09 08 58

**Comment by RonaldDijkstra** [comment]
Thanks for that.

**Pull request merged**